src/syft/lib/torchvision/allowlist.py

```diff
@@ -12,11 +15,13 @@
 allowlist["torchvision.transforms.ToTensor"] = "torchvision.transforms.ToTensor"
 allowlist["torchvision.transforms.Normalize"] = "torchvision.transforms.Normalize"
 # TODO: Normalize properties only exists on the object not on the class?
-# allowlist["torchvision.transforms.Normalize.inplace"] = "syft.lib.python.Bool"
+allowlist["torchvision.transforms.Normalize.inplace"] = "syft.lib.python.Bool"
```

Comment:
I am having a feeling that we might want the ```allowlist``` to become a more "standalone" class such that we might add more things to it? (like we have the ```module_pargs```)
What do you think @madhavajay, @tudorcebere ? or maybe I am overthinking this?